src/map/npc_chat.cpp

```diff
@@ -5,7 +5,7 @@
 
 #include "npc.hpp"
 
-#include "../../3rdparty/pcre/include/pcre.h"
+#include <pcre.h>
```

Comment:
Just because I see it below:
Should this actually be inside a ifdef for PCRE_SUPPORT?

Reply:
the whole file is wrapped around the ifdef

Reply:
Perfect, thanks!